### PR TITLE
Update Defender format and rationales

### LINKED ISSUE #246: Adjudicate "substantive changes" to the Defender Baseline Markdown
# 💡 Summary #

This issue captures the high-level tasks required to convert the existing Defender Secure Configuration Baseline document to a new structure based on the existing markdown format.  The updates will include reordering and regrouping of existing policy items, additional fields for each policy item, and policy changes to clarify relative to existing implementation guidance.

## Motivation and context ##

The new baseline policy document format will provide additional clarity to readers and support better automated assessment of baseline policy items for Defender.

## Implementation notes ##

In order to make the changes referenced above, the following tasks will need to be accomplished:

- [x] CREATE and NUMBER policy group statements based on existing policy areas using updated markdown files found in #212 and its associated branch 
   - [x] REMOVE “SHALL” and “SHOULD” language from new policy group statements
   - [x] RENAME policy groups to cover associated policies (Naming should be descriptive vice prescriptive)
   - [x] REGROUP baseline policies that are better orientated to the new Group Name if needed
   - [x] CREATE short 1-2 sentence summary for each policy group

- [x] #427
    _Note: Suggested format for Rationale_
    - 1st sentence: Security risk if policy is not implemented.
    - 2nd sentence: How this policy minimizes this risk.

- [x] UPDATE policy item description to match the desired best practice and synchronize with implementation steps (e.g., setting should be 30 days or less when implementation shows setting days to <= 30)

## Acceptance criteria ##

How do we know when this work is done?

- [x] Defender baseline markdown has been updated with new policy groups and all policy items reordered into those groups
- [x] All Defender baseline policy groups have been numbered
- [x] Each Defender policy group contains a concise and clear 1-2 sentence summary
- [x] Each Defender policy item contains a `Rationale` section using the suggested format
- [x] Each Defender policy item has been reviewed and updated to synchronize with the implementation steps

### LINKED ISSUE #427: Defender - Add Rationale section and text for each policy
Add rationales for each of the policy items in the Defender baseline using the preferred style and format.  Deconflict from existing policy group descriptions.

### PR DESCRIPTION
**Pull request by schrolla**
#  #

## 🗣 Description ##

Update the Defender baseline markdown to use the latest format and style as well as add rationales for the remaining policy items that are currently listed as TODO.

## 💭 Motivation and context ##

The current version of the Defender baseline has been reworked to use major formatting points like policy IDs and groups, but some of the smaller format and style changes are still lacking.  Additionally, the previously existing policies do not yet have rationales associated with each policy item.  Policy items will be updated to add those rationales to remove TODO placeholders.

Closes #246
Closes #427 

## 🧪 Testing ##

Confirm that the Defender related markdown checks in the GitHub action pass.
Compare the Defender baseline to the format example document.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing markdown check tests pass.

## ✅ Pre-merge checklist ##

- [x] Branch has been rebased against emerald and no conflicts remain

## ✅ Post-merge checklist ##

- [ ] Confirm markdown tests for Defender are passing in emerald branch

### CODE REVIEW

File: baselines/defender.md
Comment (ahuynhMITRE) [inline baselines/defender.md:1]: "MAY" policy to be updated in MS.DEFENDER.2.4v1

Comment (ahuynhMITRE) [reply baselines/defender.md:1]: yep perfect!

File: baselines/defender.md [hunk 1]
```diff
@@ -502,87 +560,116 @@ real-time insight into possible security incidents.
 
 ### Policies
 #### MS.DEFENDER.5.1v1
-At a minimum, the alerts required by the *Exchange Online Minimum Viable Secure Configuration Baseline* SHALL be enabled.
-- _Rationale:_ TODO
+At a minimum, the alerts required by the _Exchange Online Minimum Viable Secure Configuration Baseline_ SHALL be enabled.
```
[the remark this answers is not in the document]
Comment (ahuynhMITRE) [reply baselines/defender.md]: nothing else from me!

**Review by ahuynhMITRE — APPROVED** [review]
earlier comments have been addressed!

File: baselines/defender.md [hunk 2]
```diff
@@ -186,10 +287,9 @@ Safe attachments SHOULD be enabled for SharePoint, OneDrive, and Microsoft Teams
...
 Teams, follow the instructions listed at [Turn on Safe Attachments for
 SharePoint, OneDrive, and Microsoft Teams \| Microsoft
-Docs](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/turn-on-mdo-for-spo-odb-and-teams?view=o365-worldwide).
+Learn](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/safe-attachments-for-spo-odfb-teams-configure?view=o365-worldwide).
 
-1.  Sign in to [Microsoft 365
-    Defender](https://security.microsoft.com/).
+1.  Sign in to **Microsoft 365 Defender**.
 
 2.  Under **Email & collaboration**, select **Policies & rules**.
```
[the remark this answers is not in the document]
Comment (nanda-katikaneni) [reply baselines/defender.md:295]: Sounds good.

**Review by nanda-katikaneni — APPROVED** [review]
Reviewed changes - looks good. Approving.

**Pull request merged**